google-cloud-compute/pom.xml

```diff
@@ -35,6 +35,10 @@
       <groupId>com.google.api</groupId>
       <artifactId>gax</artifactId>
     </dependency>
+    <dependency>
+      <groupId>com.google.api</groupId>
+      <artifactId>gax-grpc</artifactId>
```

Comment:
Can you add source code comment explaining how it helps native image compilation? Is it for `com.google.api.gax.grpc.nativeimage.ProtobufMessageFeature`.

Reply:
Good call, done.